tuning/utils/preprocessing_utils.py

```diff
@@ -86,11 +149,22 @@ def get_data_collator(
             Response template to be used for formatting by TRL.
         tokenizer: AutoTokenizer
             Loaded tokenizer object to be used by the collator.
+        max_sequence_length: int
+            Max sequence length to be used for sequence tokenization.
+        data_args: configs.DataArguments
+            training data arguments
 
     Returns:
         Callable
             Callable collator to be leveraged by the trainer.
     """
+    if data_args.is_pretokenized:
+        return DataCollatorForSeq2Seq(
+            tokenizer=tokenizer,
+            padding="max_length",
```

Comment:
@kmehant can we handle the datacollator kwargs more carefully, because this collator calls `pad()` underneath, and there are various options 

https://github.com/huggingface/transformers/blob/ef177a5e1cdf0ca53e24e6d76e813198f7300dc4/src/transformers/tokenization_utils_base.py#L3418-L3429

If we decide for the user what options, then we need to chose the best one, for example we probably do not want to chose `padding='max_length'` aor `padding=None`

Reply:
And if there are defaults, then we can see below https://github.com/foundation-model-stack/fms-hf-tuning/pull/272/files#diff-0949af2da3611757d2552841654a34f1fe0a349ee114ce6342b9e0ade3eb13a6R180 how they are handled in `configs`

Reply:
Thanks, updated!

Reply:
> On the fence on "pt" or configs.PADDING_RETURN_TENSORS


Avoided this since collator should mostly return tensors rather python lists.